src/v2/kubernetes_cluster/spec/cluster.rs

```diff
@@ -278,7 +278,8 @@ impl Cluster {
         Action {
             precondition: |input: (Option<Message>, Option<ObjectRef>), s: ClusterState| {
                 &&& self.controller_models.contains_key(controller_id)
-                &&& received_msg_destined_for(input.0, HostId::Controller(controller_id))
+                &&& input.1.is_Some()
+                &&& received_msg_destined_for(input.0, HostId::Controller(controller_id, input.1.get_Some_0()))
```

Comment:
Is it true that input.1 should ALWAYS be Some here (for the state machine to take next action)?